plugins/GSdx/res/tfx.fx

```diff
@@ -572,6 +572,9 @@ float4 ps_color(PS_INPUT input)
 {
 	datst(input);
 
+#if PS_CHANNEL_FETCH == 1
+	float4 t = fetch_blue(int2(input.p.xy));
+#endif
```

Comment:
Still needs the else here and #endif after line 578. ;)

Reply:
Well that's 1 crash fixed :weary:

Still 1 left tho.